### PR TITLE
Update harvester-network-controller chart

### PR DESCRIPTION
**Pull request by yaocw2020**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**

need to bump harvester network controller chart 

**Solution:**

bump harvester network controller chart 

**Related Issue:**
#369 

**Test plan:**

none

### CODE REVIEW

File: deploy/charts/harvester/dependency_charts/harvester-network-controller/values.yaml [hunk 1]
```diff
@@ -26,3 +26,43 @@ tolerations:
...
 
 affinity: {}
+
+
+# Default value for harvester-network-controller-manager deployment
+replicas: 3
+
+strategy:
+  rollingUpdate:
+    maxSurge: 1
```
Comment (guangbochen) [inline deploy/charts/harvester/dependency_charts/harvester-network-controller/values.yaml]: why do we intend to make the max surge of manager pod to be 1?

**Review by guangbochen — COMMENTED** [review]
you will need to run `helm dep update` to generate and add those packaged charts

File: deploy/charts/harvester/dependency_charts/harvester-network-controller/values.yaml (hunk 1 above)
Comment (yaocw2020) [reply deploy/charts/harvester/dependency_charts/harvester-network-controller/values.yaml]: It was added by @futuretea. It's optional and I will delete it.

File: deploy/charts/harvester/dependency_charts/harvester-network-controller/templates/daemonset.yaml [hunk 2]
```diff
@@ -16,14 +16,18 @@ spec:
...
         - name: network
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - harvester-network-controller
+          args:
+            - agent
           securityContext:
             privileged: true
           volumeMounts:
```
Comment (gitlawr) [inline deploy/charts/harvester/dependency_charts/harvester-network-controller/templates/daemonset.yaml:33]: The volume mounts are different from https://github.com/rancher/harvester-network-controller/blob/7dbfc4eaca8bda37e3a0adf359856f681e4a70aa/manifests/daemonset.yaml#L46

**Review by guangbochen — APPROVED** [review]
LGTM

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**